vendor/css/material-icons.css

```diff
@@ -633,6 +633,9 @@
 .mdi-data-usage:before {
   content: "\e6c6";
 }
+.mdi-date-range:before {
+  content: "\e916";
+}
 .mdi-dehaze:before {
```

Comment:
Will try to add more of these in the weekend, trying to figure out a way of expediting this.